packages/zilliqa-js-crypto/src/index.ts

```diff
@@ -9,7 +12,18 @@ const NUM_BYTES = 32;
  * @returns {string} - the hex-encoded private key
  */
 export const generatePrivateKey = (): string => {
-  return randomBytes(NUM_BYTES);
+  let privateKey: string = randomBytes(NUM_BYTES);
```

Comment:
@edisonljh @AmritKumar can you please take a look at this function and let me know if you see any potential problems. This was refactored to prevent any inconsistency resulting from endianess.